installers/rpm-code/codes/common.sh

```diff
@@ -29,6 +29,20 @@ common_python() {
         tables
     # Conflict between rscode-pyzgoubi and rscode-ml so just include here
     install_pip_install PyYAML
+    # Conflicts between rscode-rsbluesky and rscode-ml
+    install_pip_install \
+        cachetools \
+        scikit-image \
+        tifffile
+    # Conflicts between rscode-rsbluesky and rscode-rsbeams
```

Comment:
Ok. Is there a reason we document the conflict of PyYAML but not these?

Reply:
0bfaa1f

Reply:
It's fine to document. It's tricky with this type of documentation, because if bluesky, say, switches to a new type of yaml library, then the documentation is out of date, but it's not wrong to include pyyaml in common. Most of the things in common are to avoid rpm conflicts, that is, two or more codes use the package.

One thing that's probably good to do is make a single install_pip_install line. It's something [I changed yesterday](https://github.com/radiasoft/rsconf/commit/d6dd5598451e86ecdd0915c3280a7f77d100b799#diff-dba0ea4b03e562fa0a77f5119e0c0b5e6247cd8f4a6dfd1ec725c32a2d0ffbdaL106) in another context. If there are no dependencies in the order of the the statements, it's better to include it as one statement to pip (or whatever function) as it gives the function more information and reduces the number of calls. With pip this is important, because of conflict detection and resolution.